yashsachdev.HabitTracker/DisplayClass.cs

```diff
@@ -0,0 +1,242 @@
+            {
+                Console.WriteLine("You do not have a habit with that name. Please try again.");
+                return;
+            }
+            Console.WriteLine($"Enter the values you want to update");
+            string resunit = UserInput.GetUnitMeasurement();
+            string habitUnit = UserInput.GetHabitUnit(resunit);
+            var habit_id = habitEnrollRepo.GetHabitId(habitname, this.UserId);
+            habitEnrollRepo.UpdateUserHabit(this.Email, habitname, habitUnit, habit_id);
+            habitEnrollRepo.DisplayUserHabit(this.Email);
+        }
+    }
+    private void DeleteHabit()
+    {
+        HabitEnrollRepo habitEnrollRepo = new HabitEnrollRepo();
+        HabitRepo habitRepo = new HabitRepo();
+        string habitname = UserInput.GetHabitName();
+        if (!habitEnrollRepo.CheckIfHabitExistsForUser(habitname, UserId))
+        {
+            Console.WriteLine("You do not have a habit with that name. Please try again.");
+            return;
+        }
+        var habit_Id = habitEnrollRepo.GetHabitId(habitname, UserId);
+        habitRepo.DeleteHabit(habit_Id);
+        Console.WriteLine($"Deleted {habitname}");
+        Console.WriteLine("Press any key to continue...");
+        Console.ReadKey();
+    }
+    private void Logout()
+    {
+        this.LoggedIn = false;
+    }
+    private void ViewHabits()
+    {
+        HabitEnrollRepo habitEnrollRepo = new HabitEnrollRepo();
+        habitEnrollRepo.DisplayUserHabit(this.Email);
+        Console.WriteLine("Press any key to continue...");
+        Console.ReadKey();
+    }
+    private void InsertHabit()
+    {
+        string habitName = UserInput.GetHabitName();
+        if (UserInput.ValidateName(habitName) == false)
+        {
+            Console.WriteLine("Please enter valid habit");
+            return;
+        }
+
+        HabitRepo habitRepo = new HabitRepo();
+        UserRepo userRepo = new UserRepo();
+        HabitEnrollRepo habitEnrollRepo = new HabitEnrollRepo();
+        string unit = UserInput.GetUnitMeasurement();
+        string habitUnit = UserInput.GetHabitUnit(unit);
+        Habit habit = new Habit
+        {
+            Habit_Name = habitName,
+            Unit = habitUnit,
+        };
+        if (userRepo.GetIdFromEmail(this.Email) == 0)
+        {
+            Console.WriteLine("User with email '{0}' not found.", this.Email);
+            return;
+        }
+
+        this.UserId = userRepo.GetIdFromEmail(this.Email);
+        if (habitEnrollRepo.CheckIfHabitExistsForUser(habitName, this.UserId))
+        {
+            Console.WriteLine("Habit '{0}' already exists for user with email '{1}'.", habitName, Email);
+            return;
+        }
+
+        habitRepo.Save(habit);
+        userRepo = new UserRepo();
+        this.UserId = userRepo.GetIdFromEmail(this.Email);
+        this.HabitId = habitRepo.GetLastInsertedId();
+        Console.WriteLine("Habit created successfully");
```

Comment:
Again try to avoid too many lines in a row, group your statements and separate them with an empty line. Ex: Separate the Console.WriteLines from the 4 statements above.